map2loop/mapdata.py

```diff
@@ -466,7 +466,7 @@ def open_http_query(url: str):
         """
         try:
             request = urllib.Request(url, headers={"Accept-Encoding": "gzip"})
-            response = urllib.request.urlopen(request, timeout=30)
+            response = urllib.request.urlopen(request, timeout=120)
```

Comment:
What happens if the url returns a bad status code?

Reply:
@lachlangrose do you mean that it would be best to have a http/url error/exception handle?

Reply:
I mean what will happen if the request returns a 404? Is that caught by the exception or do you need to raise for status?

Reply:
At the moment if it fails for any reason the DTM is set to None and all altitudes calculated from DTM are at sealevel.  I can see why increasing the timeout might be required for large height maps however 2 minutes is quite a long time to stall the process.

This feeds into writing a better generic file retrieval class (for DTM, shapefiles, config or clut files) which was on my todo list but didn't make it to a formal issue.  I'll create the issue from this thread and we can set the priority at the next meeting.

Reply:
Issue added, are we happy with the 2 minute timeout, I'd be happier with 1 unless there are known cases that require 2.

Reply:
@RoyThomsonMonash my usual go-to is to wait a random time between 10 seconds and 1 minute. The randomness means if you call the function in two separate places from two threads you are less likely to get conflicts.

Reply:
Adding randomness is a good option for multithreaded applications to spread out network interactions and we should add that to the file retrieval class but for this MR I'm happy with a fixed timeout.

@AngRodrigues what was the case that hit this limit (a large DTM file, a network pause or something else)?

Reply:
I agree, reverted back to the original timeout in https://github.com/Loop3D/map2loop/pull/91/commits/301242f931b92ac01c4251ecf9039119d9da49d5

Will close now this thread, given that the issue has been raised.